lib/repo-data.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+
+export const repoPairs = [{
```

Comment:
If we really wanted to abstract this some more, we could make this a configuration option and pass it into an environment variable via "Settings".

Really just a question of how far we want to take this (at least at this point in time 😄).

Reply:
Opened issue https://github.com/digitalbazaar/vc-commit-tracker/issues/3